native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/tasks/BuildNativeImageTask.java

```diff
@@ -220,4 +229,28 @@ public void exec() {
         }
     }
 
+    private void checkRequiredVersionIfNeeded(File executablePath, NativeImageOptions options) {
+        if (!options.getRequiredVersion().isPresent()) {
+            return;
+        }
+        try {
```

Comment:
:x: This should use `getExecOperations()` instead, as using `ProcessBuilder` is known to have issues (in particular under Windows). Should be quite straightforward API.

Reply:
Ok done